### ISSUE TITLE
March 30, 2022, Botswana case and death data

### ISSUE THREAD

**Issue by 3dgiordano** [CSSEGISandData/COVID-19#5577]
I have found that 41576 positive cases have been suddenly reported in a single day.
Also 67 deaths on March 30.

I haven't found the official data source to corroborate, doesn't working now, but the fact that it's a sudden jump might be a possible issue.

I report it for review.

**Comment by 3dgiordano** [comment]
I can access to the dashboard and they have not been updating their data since Feb 16, 2022

The contact about the data of the dashboard is the Professor Gregory Hillhouse hillhouseg@biust.ac.bw

I found out that the Africa CDC is not reporting their data correctly, they are also stuck on data from mid-February.

But I noticed that in the reports they make on Facebook, the total number of cases and deaths is reported weekly, and from there I notice that yes, what is not reported on the dashboard is reported there.
https://www.facebook.com/BotswanaGovernment/posts/350933967078688

Perhaps it is possible to contact those who maintain the dashboard to obtain historical data so that they can reconstruct what has not been reported since mid-February.
